xml/docu_styleguide.language.xml

```diff
@@ -439,9 +439,7 @@
    Always let the reader know the objective of an action before describing
    the action itself. As an example, write: <quote>To restore world peace,
    click <guimenu>Shake hands</guimenu>.</quote>
-   <remark>sknorr 2014-01-07: it's hard to take this quote seriously.
```

Comment:
actually, I agree with Stefan. 
Maybe write an example like: "To save the settings, click 'OK'."

Reply:
I'm actually so much in favor of the 'world peace' lifehack :1st_place_medal:

Reply:
Unfortunately world peace is not that easy, so let's stick to what we can.

Reply:
You are no fun! )) I just wanted to delete an old remark here :-)

Reply:
sorry Daria, it's just the old boys scout rule ;)